src/bun.zig

```diff
@@ -1137,6 +1137,7 @@ pub const getcwd = std.os.getcwd;
 
 pub fn getcwdAlloc(allocator: std.mem.Allocator) ![]u8 {
     var temp: [MAX_PATH_BYTES]u8 = undefined;
+    @memset(temp[0..], 0);
```

Comment:
Is this necessary? getcwd expects a fixed size buffer. If it is necessary, we should initialize with `.{0} ** MAX_PATH_BYTES` because it will potentially avoid dirtying the pages

Reply:
No, it's not necessary! Thank you. This should be dependent on being a Valgrind build (as the SIMD code confuses Valgrind).